docs/getting-started.md

```diff
@@ -1,35 +1,51 @@
 ---
 id: getting-started
 title: Getting started with Front-Commerce
-sidebar_label: Getting started
+sidebar_label: Installation
 ---
 
-> **Warning:** on our road to the v1.0, we are working on
-> splitting the core into manageable npm packages and composer
-> modules.
->
-> The install process is aimed at being as idiomatic as possible
-> for developers having installed composer dependencies and started
-> JS applications (server or client)
->
-> We will update this documentation as soon as our work is done.
-
-This page will help you getting started with Front-Commerce
-for a Magento2 store. Keep in mind that working with other
-e-commerce solutions is possible, but not yet part of the core.
-[Contact us](mailto:contact@front-commerce.com) if you would like to
-use Front-Commerce with another solution.
-
-## TL;DR
-
-Installing Front-Commerce is about:
-
-* ensuring you have access to our codebase (license…)
-* installing the Front-Commerce extension in your Magento2 store
-* configuring Magento2 so it could communicate with Front-Commerce
-* initializing a new Front-Commerce project
-* run the design system
-* run the application
-* tweaking the configurations to match your needs
-
-Are you ready?
+While Front-Commerce is a [closed source product](http://developers.front-commerce.test/en/license.html), we don't want you to trust us blindly.
+
+Because of this, we created a lighter version of Front-Commerce that will make you feel how it is to work with our product. While it isn't [fully featured](#TODO-link-what-is-missing), we believe that if you enjoy working with it, you will enjoy working with Front-Commerce since we use the same [concepts](http://developers.front-commerce.test/docs/architecture-overview.html) and the same [file structure](#TODO-link-file-structure). You could even reuse most of your code within Front-Commerce if you want to upgrade to the licensed version.
```

Comment:
instead of `that will make you feel how it is` I would have said `that will make you feel what it's like`

I was trying to find a better formulation for this sentence but that's the best I could come up with